news/4461.bugfix

```diff
@@ -0,0 +1,4 @@
+This fixes an issue where when someone who tries to use git
+with pip but pip can't because git is not in the path
+environment variable. This explicitly tells them how they
+can fix the issue.
```

Comment:
Given the changed wording of the error, I'd reword this as well - something like "clarifies the error given to suggest to the user what might be wrong".